scripts/handle-user-data

```diff
@@ -2,7 +2,7 @@
 
 LABEL=b2d-data
 MAGIC="boot2docker, please format-me"
-UNPARTITIONED_HD="/dev/$(lsblk | grep disk | cut -f1 -d' ')"
+UNPARTITIONED_HD="/dev/$(lsblk | grep disk | grep ".*da" | cut -f1 -d' ')"
```

Comment:
It should be part of other commit "Disable floppy disk driver commit"

Reply:
Done